.github/workflows/build_test.yml

```
@@ -6,13 +6,15 @@ on:
   pull_request:
     paths-ignore:
       - 'workflow/docker_publish.yml'
+      - '.github/workflow/changelog_test.yml'
```

Can you PR this into my PR #1407 
but both of these lines are both wrong:
```suggestion
      - '.github/workflows/docker_publish.yml'
      - '.github/workflows/changelog_test.yml'
```